modules/sql-engine/src/test/java/org/apache/ignite/internal/sql/engine/planner/datatypes/BaseTypeCoercionTest.java

```diff
@@ -51,6 +53,14 @@ static Stream<Arguments> allNumericPairs() {
         return Arrays.stream(NumericPair.values()).map(Arguments::of);
     }
 
+    static void checkIncludesAllTypePairs(Stream<Arguments> args) {
```

Comment:
it's better to revert this change. BaseTypeCoercionTest is base class for all type pairs, not only numeric ones.

Reply:
I will just rename the method